spec/workers/metrics/record_daily_usage_measurables_worker_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe Metrics::RecordDailyUsageMeasurablesWorker, type: :worker do
+  include_examples "#enqueues_on_correct_queue", "low_priority", 1
+  let_it_be(:first_article) { create(:user, comments_count: 1) }
+  let_it_be(:first_article) { create(:user, comments_count: 2) }
+  let_it_be(:first_article) { create(:user, comments_count: 0) }
+  let_it_be(:first_article) { create(:article, score: 15, nth_published_by_author: 1) }
+  let_it_be(:first_article) { create(:article, score: 5, nth_published_by_author: 2) }
+  let_it_be(:first_article) { create(:article, score: 38, nth_published_by_author: 3) }
```

Comment:
I know we're using before all, but these having the same name is a bit confusing to the casual reader :D

Reply:
Oh yeah whoops.

We never actually use these variables defined in let. Is it possible to do this without declaring a name, just the `create` bit?

Reply:
Sure! You can use [before_all](https://test-prof.evilmartians.io/#/before_all ) (it's already activated in our test suite, though we aren't using it yet):

```ruby
  before_all do
    create(:user, comments_count: 1)
    create(:user, comments_count: 2)
    create(:user, comments_count: 0)
    create(:article, score: 15, nth_published_by_author: 1)
    create(:article, score: 5, nth_published_by_author: 2)
    create(:article, score: 38, nth_published_by_author: 3)
  end
```